### ISSUE TITLE
New coin request: $RUPX, Rupaya

### ISSUE THREAD

**Issue by bitracetam** [masternodes/vps#120]
New coin request: $RUPX, Rupaya

-https://bitcointalk.org/index.php?topic=2956408.0
-https://www.rupayacoin.org/
-https://github.com/rupaya-project/rupaya
-Binary: /usr/bin/rupayad
   Configuration file:  /etc/rupaya/rupaya.conf
   Data directory:      /var/lib/rupayad
   PID file:            /var/run/rupayad/rupayad.pid (OpenRC and Upstart)
                          /var/lib/rupayad/rupayad.pid (systemd)
-https://github.com/rupaya-project/rupaya/releases/tag/v.4.1.0

**Comment by bitracetam** [comment]
Thanks for all you do!

**Comment by spectrecoin** [comment]
I agree, RUPX is a very popular masternode coins and support would be great

**Comment by spectrecoin** [comment]
Can you please add support for RUPX and a health donation will come your way!
